### PR TITLE
Rsschool 2019 q1 cv html - task

### PR DESCRIPTION
**Pull request by moiseenokv**

### CODE REVIEW

**Review by elkinuk — APPROVED** [review]
70

File: index.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <li>Viber:
+          <a href="viber://chat?number=375293224002">+375-29-322-40-02</a>
+        </li>
+      </ul>
+    </div>
+  </header>
+  <main>
+    <div class="container">
+      <!-- Summary block begin-->
+      <div class="block summary">
```
Comment (elkinuk) [inline index.html:34]: looks more like section or article

File: style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
+* {
+  margin: 0;
+  padding: 0
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #444;
+  background: #222222;
```
Comment (elkinuk) [inline style.css]:
```suggestion
  background: #222;
```

**Review by elkinuk — APPROVED** [review]
https://cqxg.github.io/rsschool-2019Q1-cv/cv - gh-pages with markdown
Student met all requirements to Markdown and deploying
+25

Student met all requirements to Pull Request and commits structure
+50

No big mistakes in html/css
https://moiseenokv.github.io/rsschool-2019Q1-cv/ - link to the gh-pages
+50

Summary: 125

File: index.html (hunk 1 above)
Comment (moiseenokv) [reply index.html:34]: OK! Thank you! I`ll keep in mind next time

File: style.css (hunk 2 above)
Comment (moiseenokv) [reply style.css]: code fixed to correct format

**Pull request merged**